src/org/etools/j1939_84/bus/RP1210Bus.java

```diff
@@ -323,42 +285,45 @@ private Packet decode(byte[] data, int length) {
         return buf;
     }
 
-    private BusLogger getLogger() {
-        return logger;
-    }
-
     /**
      * Checks the {@link RP1210Library} for any incoming messages. Any incoming
      * messages are decoded and added to the queue
      */
     private void poll() {
         try {
             while (true) {
+                byte[] data = new byte[32];
```

Comment:
Is there any advantage to making this `final`?

Reply:
Not for performance reasons with modern Java.